.github/workflows/pkgcheck.yml

```diff
@@ -15,63 +15,71 @@ jobs:
           - name: Ubuntu GCC -m32
             os: ubuntu-latest
             compiler: gcc
-            packages: gcc-multilib
+            packages: gcc-multilib g++-multilib
             cmake-args: -DCMAKE_C_FLAGS=-m32
             cflags: -m32
             ldflags: -m32
 
-          - name: Ubuntu GCC ARM HF
+          - name: Ubuntu GCC ARM SF
             os: ubuntu-latest
-            chost: arm-linux-gnueabihf
-            compiler: arm-linux-gnueabihf-gcc
-            cmake-args: -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-arm.cmake -DCMAKE_C_COMPILER_TARGET=arm-linux-gnueabihf
-            packages: qemu gcc-arm-linux-gnueabihf libc6-dev-armhf-cross
+            cmake-args: -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain-arm.cmake
+            packages: qemu gcc-arm-linux-gnueabi g++-arm-linux-gnueabi libc-dev-armel-cross
```

Comment:
`chost:` and `compiler:`were removed here instead of correcting them.

Reply:
Right, it was changed to be similar to the ARM SF targets from cmake.yml, will fix.